### PR TITLE
creates a resources for new developers page

### LINKED ISSUE #234: Create a "Resources for Devs" Page
As we've amalgamated and produced more dev resources, we should link to them somewhere! I propose we create a new dev page that lists resources that we (Teach LA + Teach LA adjacent) have made; in particular, this includes:

* learning lab crash course
* QWER Hacks React / Firebase workshops ('21 and '20)
* react starter barebones 
* Dev Intern W'21 Training

As we produce more resources, we should add them to this list too!

### PR DESCRIPTION
**Pull request by ldalton02**
Closes #234.

### CODE REVIEW

**Review by malsf21 — CHANGES_REQUESTED** [review]
Just to sum up what we talked about at today's dev team meeting:

* I can handle the copy - don't worry about that
* if you can, please create a clear visual difference from video/written resources and template/library code!
* to fix the mobile responsiveness issues, I suggest you try using media queries and display grid; [here's an example using the event.scss](https://github.com/uclaacm/teach-la-website/blob/master/_sass/_event.scss) code

**Comment by malsf21** [comment]
Just checking in - not sure if you've seen yet @ldalton02, but I think there were some issues resolving the merge conflict and thus the site won't build. Also, is there a reason there are a lot of whitespace changes with this PR?

File: _sass/_basic.scss [hunk 1]
```diff
@@ -328,3 +328,60 @@ ul > li {
 .fill-screen-row {
     width: 100%;
 }
+
+
+.resource-sub-title {
+    text-align: center;
+}
```
Comment (mattxwang) [inline _sass/_basic.scss:335]: This is a duplicate class - we already have `.text-center`!

File: _sass/_basic.scss [hunk 2]
```diff
@@ -328,3 +328,60 @@ ul > li {
...
+    margin-right: 1em; 
+    margin-left: 1em;         
+    @media (max-width: $desktop-width) {
+        margin-left: 1em;
+    }
+}
+
+.grid-reversed {
+    grid-template-columns: 2fr 1fr ;
+}
```
Comment (mattxwang) [inline _sass/_basic.scss:366]: Do you use this class?

File: devresources.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
+---
+layout: default
+title: "Developer Resources"
+permalink: "/devresources"
+---
+
+<div class="container container-fill container-restricted">
```
Comment (mattxwang) [inline devresources.html:7]: This already gets applied by the default layout, you don't need to add it again

File: _sass/_basic.scss [hunk 4]
```diff
@@ -328,3 +328,60 @@ ul > li {
...
+.resource-sub-title {
+    text-align: center;
+}
+
+.resource-logo {
+    max-height: 200px;
+    max-width: 200px;
+    width: auto;
+    height: auto;
+    box-sizing: border-box;
```
Comment (mattxwang) [inline _sass/_basic.scss:342]: This is already set by default

File: _sass/_basic.scss [hunk 5]
```diff
@@ -328,3 +328,60 @@ ul > li {
...
+    @media (max-width: $desktop-width) {
+       height: auto;
+       width: 90%;
+   }
+}
+
+.logo {
+    box-sizing: border-box;
```
Comment (mattxwang) [inline _sass/_basic.scss:350]: Also already set my default

File: _sass/_basic.scss [hunk 6]
```diff
@@ -328,3 +328,60 @@ ul > li {
...
+    justify-content: center;
+}
+
+.resource-description {
+    align-items: left;
+    margin-right: 1em; 
+    margin-left: 1em;         
+    @media (max-width: $desktop-width) {
+        margin-left: 1em;
+    }
```
Comment (mattxwang) [inline _sass/_basic.scss:361]: This media query seems like it doesn't do anything?

**Review by mattxwang — CHANGES_REQUESTED** [review]
Hey Luke,

Thanks for resolving the merge issues. Just some thoughts:

* there seems to be a lot of extraneous CSS - is every property needed in each of these cases? I think there's quite a bit of flex-stuff in particular (ex all the `align-items`) that's not needed; the most flex you'd need is probably just a vertical centering solution.
* left some other comments on CSS duplication
* remember that for links, `"here"` is not very accessible text; instead, try using something like 

```html
Browse <a class="tla-link" href="...">the documentation<a/> for more information.
```

You may also want to use `rel="noopener noreferrer"` for external links.

Thanks for your revisions so far! After this round, I can update the copy and then we'll be good to merge.

(p.s. I may also change the path and/or the footer)

File: _sass/_basic.scss [hunk 7]
```diff
@@ -328,3 +328,43 @@ ul > li {
...
+
+.resource-card-container {
+    margin: 2em;
+    border: 2px solid #5eda12;
+    border-radius: 10px;
+}
+
+.logo-container {
+    text-align: center;
+}
```
Comment (mattxwang) [inline _sass/_basic.scss:370]: I think this is still duplicated with `.text-center`!

File: _sass/_basic.scss [hunk 8]
```diff
@@ -328,3 +328,43 @@ ul > li {
...
+    @media (max-width: $desktop-width) {
+       width: 90%;
+   }
+}
+
+.resource-description {
+    align-items: left;
+    margin-right: 1em; 
+    margin-left: 1em;         
+}
```
Comment (mattxwang) [inline _sass/_basic.scss:347]: I think this class isn't necessary - you can instead add padding to the parent container?

**Review by mattxwang — CHANGES_REQUESTED** [review]
Just a few other notes, and then I'll do the copy-pass and we'll merge!

**Review by mattxwang — COMMENTED** [review]
Thanks for the quick changes Luke! Looks good in general, I'll review the copy and then give you the go-ahead to merge once I get a chance to do that!

**Comment by mattxwang** [comment]
Note to self: also need to merge master into this to reflect #268.

**Review by mattxwang — APPROVED** [review]
Okay, made some content revisions. I think we're good to merge - Luke, you can do the honours, please squash and merge!

**Pull request merged**